Content.Server/StationEvents/Events/SolarFlare.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Radio.Components;
+using Content.Server.Radio;
+
+namespace Content.Server.StationEvents.Events
+{
+    public sealed class SolarFlare : StationEventSystem
+    {
+        public override string Prototype => "SolarFlare";
+
+        private const string affectedChannel = "Common";
+
+        private float _endAfter = 0.0f;
+        private bool _running = false;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<ActiveRadioComponent, RadioReceiveAttemptEvent>(OnRadioSendAttempt);
+        }
+
+        public override void Added()
+        {
+            base.Added();
+            _endAfter = RobustRandom.Next(120, 240);
```

Comment:
Unhardcode this once you make the config (remember to take a look at `GameRuleSystem.Configuration`, you dont have to index the prototype)
```suggestion
            _endAfter = RobustRandom.Next(endAfterMin, endAfterMax); // names subject to change
```